src/uhs/client/client-cli.cpp

```diff
@@ -281,7 +281,7 @@ auto main(int argc, char** argv) -> int {
             return -1;
         }
     } else {
-        std::cerr << "Unkown command" << std::endl;
+        std::cerr << "Unknown command" << std::endl;
```

Comment:
Sometimes output changes are API breaks.

Reply:
We don't make any guarantee of a stable API (this is a research project; and that will necessitate changes), so I'm not too worried about this.